### PR TITLE
Add bootstrap_form 

### PR DESCRIPTION
**Pull request by reyes-dev**
# 🔗 Issue
[#206](https://github.com/rubyforgood/pet-rescue/issues/206)

# ✍️ Description
Added bootstrap_form gem and installed. I replaced all standard form helpers with bootstrap_form helpers. I also removed redundant labels and, wherever custom text is used for labels, added a label attribute with the custom text to the form helpers as per bootstrap_form conventions. Added asterisks to required fields.

Running `rails assets:precompile` caused the asterisks to properly appear in required fields.

### CODE REVIEW

File: app/views/adopter_profiles/_form.html.erb
Comment (Ptrboro) [inline app/views/adopter_profiles/_form.html.erb:1]: I think conditional fields are no longer working as expected in this form.
For example, the field on the video should always be displayed if 'Yes' is selected in the previous question.

https://github.com/rubyforgood/pet-rescue/assets/16762860/23740545-c0d3-458b-9c83-b95f06547791

**Review by Ptrboro — COMMENTED** [review]
Great job @reyes-dev, building forms will be so much easier now! 😍 

I left some comments, a few things need some polishing. Let me know if you need help with any of them

**Comment by edwinthinks** [comment]
@reyes-dev can you fix up the merge conflicts and open issues for the suggestions provided by @Ptrboro ? Once the merge conflicts are fixed, LGTM!

@Ptrboro thank you for reviewing this PR. Your thorough detailed focus is appreciated!

**Comment by reyes-dev** [comment]
@edwinthinks Should each suggestion warrant a separate issue?

**Comment by edwinthinks** [comment]
> @edwinthinks Should each suggestion warrant a separate issue?

@reyes-dev either or works for me :)

**Comment by edwinthinks** [comment]
Looks like Brakeman out scanner found a syntax error somewhere in this PR. Could we fix that and then I think its merging time!

**Comment by edwinthinks** [comment]
LGTM!

**Pull request merged**